src/adapters/bidfluence.js

```diff
@@ -0,0 +1,59 @@
+const bidmanager = require('../bidmanager.js');
+const bidfactory = require('../bidfactory.js');
+const utils = require('../utils.js');
+
+var BidfluenceAdapter = function BidfluenceAdapter() {
+
+    //Callback
+    window.bfPbjsCB = function (bfr) {
+        var bidObject = null;
+        if (bfr.cpm > 0) {
+            bidObject = bidfactory.createBid(1);
+            bidObject.bidderCode = 'bidfluence';
+            bidObject.cpm = bfr.cpm;
+            bidObject.ad = bfr.ad;
+            bidObject.width = bfr.width;
+            bidObject.height = bfr.height;
+        } else {
+            bidObject = bidfactory.createBid(2);
+            bidObject.bidderCode = 'bidfluence';
+        }
+        bidmanager.addBidResponse(bfr.placementCode, bidObject);
+    };
+
+    function _callBids(params) {
+        var bfbids = params.bids || [];
+        for (var i = 0; i < bfbids.length; i++) {
+            var bid = bfbids[i];
+            call(bid);
+        }
+    }
+
+    function call(bid) {
+        var adunitId = utils.getBidIdParameter('adunitId', bid.params);
+        /* jshint ignore:start */
+        var publisherId = utils.getBidIdParameter('pubId', bid.params);
+        var reservePrice = utils.getBidIdParameter('reservePrice', bid.params);
+        var pbjsBfobj = {
+            placementCode: bid.placementCode
+        };
+        /* jshint ignore:end */
+        var s = document.createElement('script');

```

Comment:
I didn't use it because I need to set the Id of the new script element as per line 43, seems the loadscript function does not allow it.